docs/source/format/Columnar.rst

```diff
@@ -765,6 +765,68 @@ application.
 We discuss dictionary encoding as it relates to serialization further
 below.
 
+.. _run-end-encoded-layout:
+
+Run-End Encoded Layout
+-------------------------
+
+Run-End is a data representation that represents data as sequences of the
+same value, called runs. Each run is represented as a value, and an integer
+describing the index in the array where the run ends.
```

Comment:
```suggestion
Run-end encoding (REE) is a variation of run-length encoding (RLE). Run-end
encoding is well-suited for representing data containing sequences of the same
value, called runs. In run-end encoding, each run is represented as a value
and an integer giving the index in the array where the run ends.
```

Reply:
It looks like this and some other changes were reverted in the rebase

Reply:
yea, i noticed. I'm fixing it.